### ISSUE TITLE
RPCS3 won't recognise DS4 Controller (No DS4Windows or any other 3rd party controller software installed). 

### ISSUE THREAD

**Issue by sealevelcain** [RPCS3/rpcs3#10355]
I'm having issues setting up my Dualshock4 controller on RPCS3. I've selected the Dualshock4 handler on the gamepad settings window, but it still doesn't recognise my controller. The controller works fine with other games on Steam, Epic Games, etc. 

The log gives me these error messages:
E DS4: GetCalibrationData: hid_get_feature_report 0x05 failed! Reason: 0000000012204420
E DS4: check_add_device: GetCalibrationData failed!

I tried googling this problem but to no avail, and as far as I can tell, there are no issues raised with this error message on GitHub either.  Does anyone know how to fix this?

**Comment by arabek** [comment]
Stop steam before starting RPCS3 and powering on the controller first and check if it's working fine. IIRC steam interferes heavily with how raw pad is handled.

**Comment by sealevelcain** [comment]
> 
> 
> Stop steam before starting RPCS3 and powering on the controller first and check if it's working fine. IIRC steam interferes heavily with how raw pad is handled.

Thanks for the response. I haven't had steam running at the same time as RPCS3, and it doesn't run on startup either. I just used Steam/Epic Games as examples of where the controller has worked in the past. 

Weirdly enough, I just installed DS4Windows after reading that it worked for someone else, and after doing that and setting my handler to Xinput, it worked.  This is not ideal though, as DS4Windows causes some input lag, so I'd like to get it working natively without DS4W if that's possible.
